src/endpoints/mod_versions.rs

```diff
@@ -283,6 +299,29 @@ pub async fn update_version(
             return Err(ApiError::DbError);
         }
     };
+    let fetched_mod = Mod::get_one(path.id.as_str(), false, &mut transaction).await?;
+    if let Some(valid_mod) = fetched_mod {
+        let version = valid_mod.versions.last().unwrap();
```

Comment:
I don't really like this approach, this fetches all versions for a specific mod, while you only need the version that's actually being verified. Also there could be an edge case where the version being verified isn't the last one (dev has submitted 2 versions, admin accepts both), so a better approach would be to fetch the data you need by mod_version id